gapipy/models/base.py

```diff
@@ -126,7 +126,7 @@ def _set_resource_field(self, field, value):
             for v in value:
                 stub.append(self._model_cls(field)(v, stub=True))
         elif value:
-            stub = self._model_cls(field)(value, stub=True)
+            stub = self._model_cls(field)(value, self._client, stub=True)
         setattr(self, field, stub)
 
     def _set_resource_collection_field(self, field, value):
```

Comment:
> I think that's where the fundamental problem lies. Language is a query-time option, not global/constant, whereas almost all other Client settings are global.

Maybe we visit that option. The current client config is a mix of query-level configuration and client-level. Perhaps we can pass the query-level configuration down into the `Query` when its being initiated, but we reference the `current_client` global (now removed) so we can set cache, use pooling, etc.

Query-level parameters for the client would be: `api_language`, `api_proxy`, `api_root`, `application_key`